cypress/support/step-definitions/multi-action-button-steps.js

```diff
@@ -37,3 +39,8 @@ Then('Multi Action Button is expanded and contains three items', () => {
   multiActionButtonList().should('have.length', 3);
   multiActionButtonList().invoke('text').should('contain', MULTI_ACTION_BUTTON_INNER_TEXT);
 });
+
+Then('Multi Action Button in a hidden container is expanded and contains three items', () => {
+  multiActionButtonList().should('have.length', 3);
+  multiActionButtonList().invoke('text').should('contain', MULTI_ACTION_BUTTON_INNER_SHORTER_TEXT);
```

Comment:
The `text` could be inside the component but could not be visible - but that was the clue